### ISSUE TITLE
Manual Install (File Not Found)

### ISSUE THREAD

**Issue by Zakkaruu** [cryon/almost-mono-themes#3]
Hello,

Thanks for providing this theme!

I was having issues getting it activated in Emacs on Windows. By default, it cannot find the `almost-mono-themes.el` file when loading a theme. This is after setting up the `custom-theme-load-path` in the `%appdata%\.emacs file`. It will see the individual themes, but be unable to load.

The fix was to rename `almost-mono-themes.el` --> `almost-mono-themes-theme.el`.

Thought I'd share this in case anyone else was having some trouble. Love the theme!

--Zach

Edit: Was still having issues with the theme not loading upon restarting Emacs. I had to add all of the following to my `%appdata%/.emacs` file.

```
(add-to-list 'custom-theme-load-path "~/.emacs.d/themes/")
(load-file "~/.emacs.d/themes/almost-mono-themes-theme.el")
(load-theme 'almost-mono-black)
```

Change the `load-theme` to whatever theme you want to use.

**Comment by cryon** [comment]
Thank you for reporting and to provide a workaraound! I tried to reproduce it on my machine (mac osx) and it turned out that the main el-file `almost-mono-themes.el` needs to be on the load path (I thought `custom-theme-load-path` would be searched when `require`ing). 

**Here's how I got it to work:**
1. clone the repo to, let's say `~/.emacs.d/themes/almost-mono-themes/`
2. Evaluate `(add-to-list 'custom-theme-load-path "~/.emacs.d/themes/almost-mono-themes")` (or put it in your init)
3. Evaluate `(add-to-list 'load-path "~/.emacs.d/themes/almost-mono-themes")` (or put it in your init)
4. `M-x load-theme almost-mono-black RET`

I will see if I can get around the extra step of adding the theme to the load-path somehow!
